common/src/main/res/values/strings.xml

```diff
@@ -493,7 +493,7 @@
     <string name="sensor_description_last_notification">The details of the last notification. You must setup an allow list or explicitly allow all notifications to be sent.\n\nNote: Sending all notification data will result in heavy battery usage.</string>
     <string name="sensor_description_last_reboot">The date and time of the devices last reboot. The setting below will allow you to adjust the deadband in milliseconds, if you still find the value to jump incorrectly. The default value is 60000 (1 minute).</string>
     <string name="sensor_description_last_removed_notification">The details of the last removed notification. This can be any notification either cleared by the user or removed by an application. You must setup an allow list or explicitly allow all notifications to be sent.\n\nNote: Sending all notification data will result in heavy battery usage.</string>
-    <string name="sensor_description_last_update">The intent for the last update that was sent, periodic updates will show as \"SensorWorker\". Enabling the \"Add New Intent\" toggle will create 1 setting to allow you to register for a broadcast intent. The toggle will switch back to off once a new setting is created so you will need to turn it back on to save more intents. You can also clear out the setting value to remove the setting in the next update. You must restart the application after making changes to these settings to take effect.</string>
+    <string name="sensor_description_last_update">The intent for the last update that was sent, periodic updates will show as \"SensorWorker\". Enabling the \"Add New Intent\" toggle will create 1 setting to allow you to register for a broadcast intent. The toggle will switch back to off once a new setting is created so you will need to turn it back on to save more intents. You can also clear out the setting value to remove the setting in the next update. If you are on Android 13+ and you notice an intent longer works then you will need to add categories that the intent expects. To do this you will need to add each category after the intent with a \",\" repeating until there are no more categories. For example an intent with 2 categories will be: \"intent,category1,category2\" as the setting value. You must restart the application after making changes to these settings to take effect.</string>
```

Comment:
> If you are on Android 13+ and you notice an intent longer works

You forgot "no". However, this should be written for both first time users and upgrading users and will also need to account for new intents. I'm thinking something like "When you're using Android 13+ and you're not receiving all intents..."?

> then you will need to add categories that the intent expects.

The intent doesn't expect categories, but includes/sends them. The intent receiver expects categories (but let's not make it more complicated by introducing intent receivers here).

> add each category after the intent 

after the intent action

> with a \",\" repeating until there are no more categories.

Easier: "seperated by a ','."? Or would that mean something different?

Reply:
> You forgot "no". However, this should be written for both first time users and upgrading users and will also need to account for new intents. I'm thinking something like "When you're using Android 13+ and you're not receiving all intents..."?

good catch, will update




> after the intent action

I actually didn't specify `action` here because it was not used earlier. I will update the description to make it more explicit.



> Easier: "seperated by a ','."? Or would that mean something different?

👍 no I think thats good. The example immediately after this should clear up any confusion......i hope 😃

Reply:
>> then you will need to add categories that the intent expects.
>
> The intent doesn't expect categories, but includes/sends them. The intent receiver expects categories (but let's not make it more complicated by introducing intent receivers here).

Looks like you missed this comment, or didn't share why you don't agree.

Reply:
Oh I didn't think it was meant to be applied since you mentioned not to make it more complicated :)  

For our purposes I thought the expectation would be enough to tell a user that its required in order to get the intent registered.

Reply:
I understand the confusion now, let's keep it as it is :)